libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -171,38 +186,23 @@ export class SkyAgGridWrapperComponent
   }
 
   public ngOnInit(): void {
-    this.#themeSvc?.settingsChange
+    combineLatest<[boolean, SkyThemeSettingsChange | undefined]>([
+      this.#hasEditableClass.pipe(distinctUntilChanged()),
+      this.#themeSvc?.settingsChange ?? of(undefined),
```

Comment:
should the theme service even be optional? Is there a scenario where it wouldn't be provided?

Reply:
It's not necessarily provided in tests and requiring it becomes a breaking change.